src/components/index/createRequestsTableCells.tsx

```diff
@@ -72,7 +72,11 @@ function createRequestsTableCells(
         ? req.proposedPrice.toString()
         : undefined;
 
-      if (proposedPrice && unanswerable.includes(proposedPrice)) {
+      // req.state 1 is fresh request with no proposed answer, in that case always set to "-"
+      // proposed price sometimes defaults to 0 even if noone has proposed an answer yet.
+      if (req.state === undefined || req.state <= 1) {
+        proposedPrice = "-";
```

Comment:
this was an unrelated bug, requests with no proposed answers would sometimes show that there was an answer of 0